### DISCUSSION TITLE
Issue arising when closeread article not in the project root

### DISCUSSION THREAD

**Discussion by MarkHanly** [qmd-lab/closeread#95] [Q&A] [answered]
Hi @andrewpbray and @jimjam-slam, 

Loving exploring this package, thank you!

I am working on a quarto website with a mix of articles and to keep things organised my closeread files are in a subfolder `/lectures`. When I try to run my closeread .qmd I get the error below indicating the `_extensions` folder can't be found.

`ERROR: NotFound: No such file or directory (os error 2): lstat '/Users/z3509778/Library/CloudStorage/OneDrive-UNSW/Data Vis Workshop/CourseSite/lectures/_extensions/qmd-lab/closeread/closeread.css'`

The issue seems to be that the `_extensions` folder is in the root of the project and isn't being found by the .qmd in the subfolder. When I copy the `_extensions` folder from the root of the project to the `/lectures` subfolder it works, but this doesn't seem like a good long-term solution if I have various subfolders and other extensions.

Not sure if its a bug or if the issue is with my project/folder set up, or I missed a step in the closeread set up. Thanks for any pointers!

**Comment by jimjam-slam** [comment, marked as answer]
Hey Mark, great to hear from you! Are you running `quarto render` on the project as a whole, or on the individual doc? I believe that will determine where Quarto looks for the extension folder!

**Comment by MarkHanly** [comment]
Hi James,

Thanks for the quick response! Yes, I had been rendering the individual document and sure enough if I render the whole site it runs without an issue.  

This prompts a more general quarto workflow query, as rendering the whole site is relatively slow, even with cache enabled. Do you know a solution here, or what's your usual workflow? (sorry if too off-topic for this forum!)

**Reply by andrewpbray** [reply]
I tend to work in Positron or Rstudio and rely heavily on quarto preview (which is actually what gets run when you click the render button in rstudio). When called within a project, this will do a lazy render, just rendering files that have changed, so you don't have to sit patiently for the whole site to render when all you want to see is the most recent update to a single doc.

**Comment by MarkHanly** [comment]
Ah, I see, yeah I've been clicking the render button. Great tip, thank you Andrew!
